pkgs/development/python-modules/playwright/default.nix

```diff
@@ -125,9 +159,29 @@ let
       --set SSL_CERT_FILE /etc/ssl/certs/ca-bundle.crt \
       --set FONTCONFIG_FILE ${fontconfig}
   '' + lib.optionalString withFirefox ''
-    FIREFOX_REVISION=$(jq -r '.browsers[] | select(.name == "firefox").revision' $BROWSERS_JSON)
-    mkdir -p $out/firefox-$FIREFOX_REVISION/firefox
-    ln -s ${firefox}/bin/firefox $out/firefox-$FIREFOX_REVISION/firefox/firefox
+    firefoxoutdir=$out/firefox-${browser_revs.firefox}/firefox
+    mkdir -p $firefoxoutdir
+    cp -r ${upstream_firefox}/* $firefoxoutdir/
+
+    # patchelf the binary
+    wrapper="${firefox-bin}/bin/firefox"
+    binary="$(readlink -f $(<"$wrapper" grep '^exec ' | grep -o -P '/nix/store/[^"]+' | head -n 1))"
```

Comment:
This is at a fixed location? Or can we get that from the firefox attr via passthru?